sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcTest.scala

```diff
@@ -66,46 +71,30 @@ abstract class OrcTest extends QueryTest with SQLTestUtils with BeforeAndAfterAl
    */
   protected def withOrcFile[T <: Product: ClassTag: TypeTag]
```

Comment:
Can methods like this just be inlined now to use the new method? if the goal is simplifying here.

Reply:
I tried to inline it but seems can't make it work.

Reply:
If you suggest to replace the usage of withOrcFile with new method, seems to me we can keep them because they are used many places. To replace them all, there will be a tedious diff. Maybe it is not worth now.